Src/AutoFixture.NUnit3.sln

```diff
@@ -0,0 +1,23 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2012
```

Comment:
This solution file targets Visual Studio 2012. However, I've opened it with Visual Studio 2015 and everything went well.